addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1867,6 +1867,12 @@ export function splitElement(element, offset) {
         index < offset ? before.appendChild(child) : after.appendChild(child);
         index++;
     }
+    // e.g.: <p>Test/banner</p> + ENTER <=> <p>Test</p><div class="o_editor_banner>...</div><p><br></p>
```

Comment:
When a user creates a banner like in example, an issue arises where the after element appears empty without a line break `<br>`. Consequently, it becomes challenging to access the next line after the banner if needed.

Similarly, when creating a table using a similar approach, clicking at the end of the table 'type / for commands' powerbox hint will appear below the table. However, the cursor is still positioned at the end of the table, and any typing at that point inserts content outside the paragraph `(<p>)`. like, `<table></table>test<p></p>`
I have check the sanitizer and it's not adding `<br>` in those empty tags.

Reply:
Ok indeed I see that `fillEmpty` (which does what you need here) is indeed called in various places where we need it, and only for one specific case in the sanitizer. Note to self: it would be good to refactor that.
In the meantime, it's ok to leave it here but you should use the util (`fillEmpty`) rather than duplicate the logic.

Reply:
I've applied the suggested changes.